tests/lib/prepare.sh

```diff
@@ -774,37 +774,32 @@ EOF
         cd repacked-kernel
         rm -rf firmware/*
 
-        # the code below drops the modules that are not loaded on the
-        # current host, this should work for most cases, since the image will be
-        # running on the same host
-        # TODO:UC20: enable when ready
-
-        # To avoid shellcheck unused code warning, we cannot use "exit 0" to disable
-        # the module drop code. To avoid commented out code, we use a flag instead.
-        # Strip off the check when this UC20 code is enabled.
-        uc20Ready=false
-
-        if [ "$uc20Ready" = "true" ]; then
-            # drop unnecessary modules
-            awk '{print $1}' <  /proc/modules  | sort > /tmp/mods
-            #shellcheck disable=SC2044
-            for m in $(find modules/ -name '*.ko'); do
-                noko=$(basename "$m"); noko="${noko%.ko}"
-                if echo "$noko" | grep -f /tmp/mods -q ; then
-                    echo "keeping $m - $noko"
-                else
-                    rm -f "$m"
-                fi
-            done
-            #shellcheck disable=SC2010
-            kver=$(ls "config"-* | grep -Po 'config-\K.*')
-
-            # depmod assumes that /lib/modules/$kver is under basepath
-            mkdir -p fake/lib
-            ln -s "$PWD/modules" fake/lib/modules
-            depmod -b "$PWD/fake" -A -v "$kver"
-            rm -rf fake
-        fi
+        # The code below drops the unnecessary modules that are not loaded on the
+        # current host, this works for most cases, since the image will be
+        # running on the same host, however some additional modules are required
+
+        # Load additional required modules on current host
```

Comment:
Thanks for this comment, I would expand it a little bit though. Right now it mostly describes "what" is done, but the "why" is the important part here I feel. So maybe something like: `Load additional modules that will be used during the tests`?

Reply:
Thanks, good point. Improved comment to give clear indication of the why.